assets/queries/cloudFormation/user_missing_group/test/positive.yaml

```diff
@@ -30,6 +30,35 @@ AWSTemplateFormatVersion: "2010-09-09"
 Description: A sample template
```

Comment:
Third sample is invalid due to Groups null value in line 64:

`[cfn-lint] E0000: Null value at line 7 column 16`

Reply:
Understood. Removed that sample.